Files/Properties.xaml

```diff
@@ -6,139 +6,220 @@
+                            <SolidColorBrush x:Key="YourHomeCardBackgroundColor" Color="#f3f1ef" />
+                            <SolidColorBrush x:Key="YourHomeCardBorderColor" Color="#f3f1ef" />
+                            <SolidColorBrush x:Key="CustomInputFieldBorderBrush" Color="DarkGray" />
+                            <SolidColorBrush x:Key="RibbonBackgroundColor" Color="White" />
+                            <StaticResource x:Key="NavigationViewTopPaneBackground" ResourceKey="RibbonBackgroundColor" />
+                        </ResourceDictionary>
+                        <ResourceDictionary x:Key="Dark">
+                            <AcrylicBrush
+                                x:Key="BackgroundAcrylicBrushProperties"
+                                Windows10version1903:TintLuminosityOpacity="0.75"
+                                BackgroundSource="HostBackdrop"
+                                FallbackColor="{StaticResource SystemListLowColor}"
+                                TintColor="Black"
+                                TintOpacity="0.7" />
+                            <SolidColorBrush x:Key="NavigationViewDefaultPaneBackground" Color="#191919" />
+                            <SolidColorBrush x:Key="NavigationViewExpandedPaneBackground" Color="#191919" />
+                            <SolidColorBrush x:Key="ApplicationPageBackgroundThemeBrush" Color="#191919" />
+                            <SolidColorBrush x:Key="UnmodifiedSystemPageColor" Color="Black" />
+                            <StaticResource x:Key="YourHomeCardBackgroundColor" ResourceKey="TextControlBackground" />
+                            <SolidColorBrush x:Key="YourHomeCardBorderColor" Color="White" />
+                            <SolidColorBrush x:Key="TabViewBackground" Color="Black" />
+                            <SolidColorBrush x:Key="TabViewItemHeaderBackgroundSelected" Color="#191919" />
+                            <SolidColorBrush x:Key="TabViewItemHeaderBackgroundPointerOver" Color="#191919" />
+                            <SolidColorBrush x:Key="TabViewItemHeaderBackgroundPressed" Color="#191919" />
+                            <SolidColorBrush x:Key="CustomInputFieldBorderBrush" Color="#FF363636" />
+                            <SolidColorBrush x:Key="RibbonBackgroundColor" Color="#131313" />
+                            <StaticResource x:Key="NavigationViewTopPaneBackground" ResourceKey="RibbonBackgroundColor" />
+                        </ResourceDictionary>
+                        <ResourceDictionary x:Key="HighContrast">
+                            <!--  This empty dictionary ensures that the default high contrast resources are used when the user turns on high contrast mode.  -->
+                        </ResourceDictionary>
+                    </ResourceDictionary.ThemeDictionaries>
+                </ResourceDictionary>
+            </ResourceDictionary.MergedDictionaries>
+        </ResourceDictionary>
+    </Page.Resources>
```

Comment:
What is this template being used for?

Reply:
For acrylic background of Properties window

Reply:
Can you just use something like `SystemControlAcrylicWindowBrush`?

Reply:
After setting Background="{ThemeResource SystemControlAcrylicWindowBrush}" there is no any acrylic blur, it is not working.

Reply:
That is strange, I think we should try to figure out the cause before adding what really shouldn't be necessary here.